src/wp-includes/post.php

```diff
@@ -8073,7 +8073,8 @@ function get_available_post_mime_types( $type = 'attachment' ) {
 		$mime_types = $wpdb->get_col( $wpdb->prepare( "SELECT DISTINCT post_mime_type FROM $wpdb->posts WHERE post_type = %s", $type ) );
```

Comment:
Empty `post_mime_type` values could probably also be filtered at the SQL query level too:
```suggestion
		$mime_types = $wpdb->get_col( $wpdb->prepare( "SELECT DISTINCT post_mime_type FROM $wpdb->posts WHERE post_type = %s AND post_mime_type <> ''", $type ) );
```

Reply:
This suggestion does more clearly convey that we don't want empty MIMEs returned. The subsequent `array_filter( $mime_types )` overlaps/reinforces this, but encompasses the pre-filter's return value as well.

Updated in f79ce9f47c.